src/ClientModel/CaptureData.cpp

```diff
@@ -111,6 +111,37 @@ void CaptureData::UpdateFunctionStats(uint64_t instrumented_function_id, uint64_
   }
 }
 
+void CaptureData::OnCaptureComplete(std::vector<orbit_client_data::TimerChain*> chains) {
```

Comment:
I'd suggest adding `ORBIT_SCOPE_FUNCTION;` at the top of this function so that is shows up in the introspection window (enabled when launching with `--devmode`). This way we get an idea of how long it takes to execute. For very long captures it could take a while, we should keep an eye on execution time.

Reply:
done